CIP-0068/README.md

```diff
@@ -329,6 +329,92 @@ We want to bring the metadata of the RFT `d5e6bf0500378d4f0da4e8dde6becec7621cd8
 3. Reference the output in the transaction. (off-chain)
 4. Verify validity of datum of the referenced output by checking if policy ID of `reference NFT` and `user token` and their asset names without the `asset_name_label` prefix match. (on-chain)
 
+#### 500 Royalty Standard
+
+The fourth introduced standard is the `500` Royalty NFT standard with the registered `asset_name_label` prefix value
+
+| asset_name_label            | class        | description                                                          |
+| --------------------------- | ------------ | -------------------------------------------------------------------- |
+| 500                         | NFT          | Royalty NFT stored in a UTxO containing a datum with royalty information |
+
+##### Class
+
+The `royalty NFT` is an NFT (non-fungible token).
+
+##### Pattern
+
+The `royalty NFT` **must** have an identical `policy id` as the collection.
+
+The `asset name` **must** be `001f4d70526f79616c7479` (hex encoded), it contains the [CIP-0067](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0067/README.md) label `500` followed by the word "Royalty".
+
+Example:\
+`royalty NFT`: `(500)Royalty`\
+`reference NFT`: `(100)Test123`
+
+##### Metadata
+
+The royalty info datum is specified as follows (CDDL):
+
+```cddl
+big_int = int / big_uint / big_nint
+big_uint = #6.2(bounded_bytes)
+big_nint = #6.3(bounded_bytes)
+
+optional_big_int = #6.121([big_int]) / #6.122([])
+
+royalty_recipient = #6.121([
+              address,                    ; definition can be derived from: 
+                                          ; https://github.com/input-output-hk/plutus/blob/master/plutus-ledger-api/src/PlutusLedgerApi/V1/Address.hs#L31
+              int,                        ; variable fee ( calculation: ⌊1 / (fee / 10)⌋ ); integer division with precision 10
+              optional_big_int,           ; min fee (absolute value in lovelace)
+              optional_big_int,           ; max fee (absolute value in lovelace)
```

Comment:
Are the min and max fee optional or are they required? If so, should there be a note that if you do not wish to set them then these values should be set to 0?

Reply:
They are optional. `optional_big_int` is defined in CDDL above this, but perhaps it should be expressed in a less niche format as well.

They are both expected to be a bigint or not be included. Max fee being set to 0 to represent no max fee would be confusing, as it could easily be misread to mean no fees at all, i.e. if the implementer of the protocol missed that wrinkle.

Reply:
Derp yeah, I guess I didn't read the names of the variables very well, but I would still add the `?` to indicate in the CDDL that they are optional :)